src/responsibleai/rai_analyse/_rai_insight_data.py

```diff
@@ -0,0 +1,475 @@
+from responsibleai import RAIInsights
```

Comment:
A thought, but if you have your own environment, should all the component code be in a different directory to `rai_analyse`?

Reply:
I expect to share the execution environment (let me know if you mean something else) once the pip dependency issue is resolved.